scripts/stack/.env

```diff
@@ -0,0 +1,3 @@
+ELASTICSEARCH_VERSION=8.11.2
+KIBANA_VERSION=8.11.2
+CONNECTORS_VERSION=8.11.2.0
```

Comment:
Do we want to use `SNAPSHOT` here?

Reply:
Probably best -- I had used 8.11 to test with, but for the main - that's a good idea... I was also thinking about tying this to the ./connectors/VERSION file as well